builder/amazon/common/access_config.go

```diff
@@ -65,8 +66,10 @@ func (c *AccessConfig) Config() (*aws.Config, error) {
 // the region from the instance metadata if possible.
 func (c *AccessConfig) Region() (string, error) {
 	if c.RawRegion != "" {
-		if valid := ValidateRegion(c.RawRegion); valid == false {
-			return "", fmt.Errorf("Not a valid region: %s", c.RawRegion)
+		if c.SkipValidation == false {
```

Comment:
The second part of this suggestion would lead to running the validation and then ignoring it.  While today that is safe as the validation has no side effects that might not be true when the next code edit comes along.  So I would prefer only make the minimal change to !c.SkipValidation as you suggest.  I'll do that and re-submit.